src/net/sourceforge/kolmafia/request/RelayRequest.java

```diff
@@ -1504,6 +1504,11 @@ private boolean sendMohawkWigWarning() {
       return false;
     }
 
+    // If they can't equip Wig, no problem
```

Comment:
As a minor note, I would prefer to see a warning that an item I temporarily can't equip could speed up a zone. I could then respond to that by deciding to be able to equip hats, for example.

Reply:
That's an interesting idea. Note that there are several reasons one might not be able to equip the item:

- Moxie less than 55 (Not a problem for my AT at level 10, but in You, Robot, one's off-stats will heavily lag one's main stat unless you intentionally spend energy to improve them.)
- You don't have the Mannequin Head equipped - something you can fix with 15 scrap.

Along the same line, one might think that the warning for equipping an antique machete for fighting lianas would have the same issue. Except, I notice that sendMacheteWarning doesn't even check whether you can equip the machete - which is why I learned to make a point up upgrading muscle to at least 62 before starting to open the zones in the Hidden City.

I'm now thinking, why not write a unit test package for RelayRequest "warnings"? That will allow me to work on your suggestion without using, say, debug prints and such. Also my "free king warning" issue.

OK. I'll work on that. Next PR. :)